OneCryptoPkg/Include/Library/BaseCryptCrtLib.h

```diff
@@ -0,0 +1,145 @@
+/** @file
+  BaseCryptCrtLib.h
+  
+  Library class for base crypto CRT support. This library provides CRT functionality
+  for cryptographic operations by managing dependency injection for memory allocation,
+  time services, random number generation, and debugging.
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef BASE_CRYPT_CRT_LIB_H_
+#define BASE_CRYPT_CRT_LIB_H_
+
+#include <Uefi.h>
+#include <Library/PrintLib.h>
```

Comment:
This one I agree with Sean's point of not including Uefi.h, because we need to be extremely careful with what this file allows. PrintLib.h should be fine, but also probably not actually needed?